pygubudesigner/widgets/entryvalidatecommandeditor.py

```diff
@@ -15,38 +15,40 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import unicode_literals
-from collections import OrderedDict
+
 import json
+from collections import OrderedDict
 
 try:
     import tkinter as tk
     import tkinter.ttk as ttk
-except:
+except BaseException:
```

Comment:
```suggestion
except ImportError:
```

idk Pygubu why even supports Python 2